docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -112,8 +112,9 @@ Alpha features might be changed or removed without prior notice.
 
 The following toggles require explicitly setting Grafana's [app mode]({{< relref "../_index.md/#app_mode" >}}) to 'development' before you can enable this feature toggle. These features tend to be experimental.
 
-| Feature toggle name | Description                                         |
-| ------------------- | --------------------------------------------------- |
-| `k8s`               | Explore native k8s integrations                     |
-| `entityStore`       | SQL-based entity store (requires storage flag also) |
-| `nestedFolders`     | Enable folder nesting                               |
+| Feature toggle name   | Description                                                    |
+| --------------------- | -------------------------------------------------------------- |
+| `k8s`                 | Explore native k8s integrations                                |
+| `entityStore`         | SQL-based entity store (requires storage flag also)            |
+| `nestedFolders`       | Enable folder nesting                                          |
+| `externalServiceAuth` | Starts an OAuth2 authentication provider for external services |
```

Comment:
Can we rephrase it to say something like "Enables an OAuth2 authentication provider for external services"?

I think we should also introduce a config parameter eventually which specifies whether the OAuth2 server is turned on or not and those two (the config and the ff) should work jointly. Wdyt?